invenio_vocabularies/contrib/names/jsonschemas/names/name-v2.0.0.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Same comment as for the records mapping, if this is not needed to update I would avoid updating only the version. To be checked.

Reply:
Yes, not needed, removed!